tsconfig.json

```diff
@@ -1,19 +1,20 @@
 {
     "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
+        "target": "ES2022",
+        "module": "Node16",
+        "moduleResolution": "Node16",
 	    "strict": true,
-    	"esModuleInterop": true,
+        "esModuleInterop": true,
     	"skipLibCheck": true,
 	    "rootDir": "./",
-        "outDir": "./dist/",
+        "outDir": "./dist",
         "allowJs": true
     },
```

Comment:
Do we also need `"lib": ["es2023"]`?

> Specify a set of bundled library declaration files that describe the target runtime environment.

Sources: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping & https://www.npmjs.com/package/@tsconfig/node20

Reply:
It's only necessary if the given codebase leverages features introduced in ES2023.